static/sitemap_test.xml

```diff
@@ -0,0 +1,134 @@
+      </url>
+      <url>
+          <loc>https://www.operationcode.org/jobs</loc>
+          <lastmod>Tue Jul 09 2019 23:14:50 GMT+0530 (India Standard Time)</lastmod>
+          <priority>0.80</priority>
+      </url>
+      <url>
+          <loc>https://www.operationcode.org/join</loc>
+          <lastmod>Tue Jul 09 2019 23:14:50 GMT+0530 (India Standard Time)</lastmod>
+          <priority>0.80</priority>
+      </url>
+      <url>
+          <loc>https://www.operationcode.org/leadership_circle</loc>
+          <lastmod>Tue Jul 09 2019 23:14:50 GMT+0530 (India Standard Time)</lastmod>
+          <priority>0.80</priority>
+      </url>
+      <url>
+          <loc>https://www.operationcode.org/login</loc>
+          <lastmod>Tue Jul 09 2019 23:14:50 GMT+0530 (India Standard Time)</lastmod>
+          <priority>0.80</priority>
+      </url>
+      <url>
+          <loc>https://www.operationcode.org/password_reset/confirm</loc>
+          <lastmod>Tue Jul 09 2019 23:14:50 GMT+0530 (India Standard Time)</lastmod>
+          <priority>0.80</priority>
+      </url>
+      <url>
+          <loc>https://www.operationcode.org/password_reset</loc>
+          <lastmod>Tue Jul 09 2019 23:14:50 GMT+0530 (India Standard Time)</lastmod>
+          <priority>0.80</priority>
+      </url>
+      <url>
+          <loc>https://www.operationcode.org/podcast</loc>
+          <lastmod>Sun Jul 14 2019 14:53:00 GMT+0530 (India Standard Time)</lastmod>
+          <priority>0.80</priority>
+      </url>
+      <url>
+          <loc>https://www.operationcode.org/policy</loc>
+          <lastmod>Tue Jul 09 2019 23:14:50 GMT+0530 (India Standard Time)</lastmod>
+          <priority>0.80</priority>
+      </url>
+      <url>
+          <loc>https://www.operationcode.org/press</loc>
+          <lastmod>Tue Jul 09 2019 23:14:50 GMT+0530 (India Standard Time)</lastmod>
+          <priority>0.80</priority>
+      </url>
+      <url>
+          <loc>https://www.operationcode.org/sponsorship</loc>
+          <lastmod>Tue Jul 09 2019 23:14:50 GMT+0530 (India Standard Time)</lastmod>
+          <priority>0.80</priority>
+      </url>
+      <url>
+          <loc>https://www.operationcode.org/team</loc>
+          <lastmod>Tue Jul 09 2019 23:14:50 GMT+0530 (India Standard Time)</lastmod>
+          <priority>0.80</priority>
+      </url>
+      <url>
+          <loc>https://www.operationcode.org/terms</loc>
+          <lastmod>Tue Jul 09 2019 23:14:50 GMT+0530 (India Standard Time)</lastmod>
+          <priority>0.80</priority>
+      </url>
+      <url>
+          <loc>https://www.operationcode.org/testing</loc>
+          <lastmod>Sun Jul 14 2019 15:45:12 GMT+0530 (India Standard Time)</lastmod>
+          <priority>0.80</priority>
+      </url>
+      <url>
+          <loc>https://www.operationcode.org/who_we_serve</loc>
+          <lastmod>Tue Jul 09 2019 23:14:50 GMT+0530 (India Standard Time)</lastmod>
+          <priority>0.80</priority>
+      </url>
+</urlset>
```

Comment:
sitemap_test.xml is a testing file. we can rename it to the original sitemap.xml file after code review.